libcnb-data/src/buildpack/mod.rs

```diff
@@ -0,0 +1,288 @@
+mod api;
+mod id;
```

Comment:
I couldn't decide whether to use `api` / `id` or `buildpack_api` , / `buildpack_id` as the internal module name, given the parent module is `buildpack` so the latter seems duplicative.